lib/langfuse_sdk/support/client.ex

```diff
@@ -40,18 +44,18 @@ defmodule LangfuseSdk.Support.Client do
   end
 
   defp execute_request(%{method: :post} = opts) do
-    request = [
-      url: build_endpoint(opts.url),
-      body: encode_body(opts[:body]),
-      retry: :transient
-    ]
-    |> Req.new()
-    |> Auth.put_auth_headers()
-    |> Req.post()
+    request =
+      [
+        url: build_endpoint(opts.url),
+        body: encode_body(opts[:body]),
+        retry: :transient
+      ]
+      |> Req.new()
+      |> Auth.put_auth_headers()
+      |> Req.post()
```

Comment:
@nickgnd was this an accident or the formatted?

Reply:
mix format on autosave :/